core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,13 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Closes the Iterator. This must be overridden by the implementing class that has access to <tt>SortedKeyValueIterator source</tt> provided in the
```

Comment:
This should say "should be overridden". Or it should say that it "is expected that". Clearly it's not a "must" situation, since the default method does exist, and allows them to avoid it. This could cause problems mixing legacy iterators with new iterators, as the new ones will be expected to be closed, but the legacy ones won't close them.